lib/node_modules/@stdlib/stats/incr/wstdev/lib/main.js

```diff
@@ -0,0 +1,131 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var isNumber = require( '@stdlib/assert/is-number' ).isPrimitive;
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var format = require( '@stdlib/string/format' );
+
+
+// MAIN //
+
+/**
+* Returns an accumulator function which incrementally computes a weighted standard deviation.
+*
+* @param {number} [mean] - known mean value
+* @throws {TypeError} must provide a number primitive for mean
+* @returns {Function} accumulator function
+*
+* @example
+* var accumulator = incrwstdev();
+*
+* var s = accumulator();
+* // returns null
+*
+* s = accumulator( 2.0, 1.5 );
+* // returns 0.0
+*
+* s = accumulator( -5.0, 2.0 );
+* // returns updated weighted standard deviation
+*
+* s = accumulator();
+* // returns current weighted standard deviation
+*
+* @example
+* var accumulator = incrwstdev( 3.14 );
+*/
+function incrwstdev( mean ) {
+    var delta;
```

Comment:
We use TAB indentation, not space. Please ensure you have EditorConfig setup in your IDE.